platform.py

```diff
@@ -59,9 +59,13 @@ def configure_default_packages(self, variables, targets):
                 if not IS_WINDOWS:
                     self.packages["tool-gperf"]["optional"] = False
 
-            if board in ("nano33ble", "nicla_sense_me"):
+            if board in ("nano33ble", "nicla_sense_me", "xiaoblesense", "xiaoble"):
```

Comment:
I think it would be better to separate the xiaoblense and xiaoble  boards in another if statement.

Reply:
Because of? As I see it, they have the same MCU, the "sense" version just has more sensors on it. What would be your suggested code change?

Reply:
It is true that they use the same MCU, but I think that since they are different boards with modified packages (framework-arduino-mbed-seeed) it would be better to separate them, but I don't know very well. I'm a random person on the internet, I don't have much idea about this.

```py
if board in ("nano33ble", "nicla_sense_me"):
    self.packages["toolchain-gccarmnoneeabi"]["version"] = "~1.80201.0"
    self.packages["tool-adafruit-nrfutil"]["optional"] = False
    self.frameworks["arduino"]["package"] = "framework-arduino-mbed"
    self.frameworks["arduino"][
        "script"
    ] = "builder/frameworks/arduino/mbed-core/arduino-core-mbed.py"

if board in ("xiaoblesense", "xiaoble"):
    self.packages["toolchain-gccarmnoneeabi"]["version"] = "~1.80201.0"
    self.packages["tool-adafruit-nrfutil"]["optional"] = False
    self.frameworks["arduino"]["package"] = "framework-arduino-mbed-seeed"
    self.frameworks["arduino"][
        "script"
    ] = "builder/frameworks/arduino/mbed-core/arduino-core-mbed.py"
```

Reply:
Ahh now I see what yo umean. But yes, this distinction is already made below
```py
                if board in ("xiaoblesense", "xiaoble"):
                    self.frameworks["arduino"]["package"] = "framework-arduino-mbed-seeed"
                else:
                    self.frameworks["arduino"]["package"] = "framework-arduino-mbed"
```
and since the rest of the packages are the same I find it to be okay to share the rest of the code (setting toolchain version, uploader and builder script).